docker-compose.yml

```diff
@@ -40,6 +40,26 @@ services:
       - "8086:8086"
     entrypoint: "gcloud beta emulators bigtable start"
 
+  learning_service:
```

Comment:
Would be good to split the `docker-compose.yml` into two separate Compose files, like in WDM repository people who develop the application usually run it locally from the IDE and not through the Compose file. Doing it this way will run the apps and occupy the ports.

Reply:
Ok. Will create separate files.